src/pykeen/datasets/codex.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+
+"""CoDEx datasets.
+
+- GitHub Repository: https://github.com/tsafavi/codex
+- Paper: https://arxiv.org/pdf/2009.07810.pdf
+
+Citation:
+
+.. [safavi2020] Safavi, T. & Koutra, D. (2020). `CoDEx: A Comprehensive Knowledge Graph
+   Completion Benchmark <http://arxiv.org/abs/2009.07810>`_.  *arXiv* 2009.07810.
+"""
+
+from .base import UnpackedRemoteDataSet
+
+BASE_URL = 'https://raw.githubusercontent.com/tsafavi/codex/master/data/triples/'
+SMALL_VALID_URL = f'{BASE_URL}/codex-s/valid.txt'
+SMALL_TEST_URL = f'{BASE_URL}/codex-s/test.txt'
+SMALL_TRAIN_URL = f'{BASE_URL}/codex-s/train.txt'
+
+MEDIUM_VALID_URL = f'{BASE_URL}/codex-m/valid.txt'
+MEDIUM_TEST_URL = f'{BASE_URL}/codex-m/test.txt'
+MEDIUM_TRAIN_URL = f'{BASE_URL}/codex-m/train.txt'
+
+LARGE_VALID_URL = f'{BASE_URL}/codex-l/valid.txt'
+LARGE_TEST_URL = f'{BASE_URL}/codex-l/test.txt'
+LARGE_TRAIN_URL = f'{BASE_URL}/codex-l/train.txt'
+
+
+class CoDeXSmall(UnpackedRemoteDataSet):
+    """The CoDeX small dataset."""
+
+    def __init__(self, create_inverse_triples: bool = False, **kwargs):
+        """Initialize the CoDeX small dataset.
+
+        :param create_inverse_triples: Should inverse triples be created? Defaults to false.
+        :param kwargs: keyword arguments passed to :class:`pykeen.datasets.base.UnpackedRemoteDataSet`.
+        """
+        kwargs.setdefault('stream', False)
```

Comment:
why default to `False`?

Reply:
hahaha github doesn't want to let you download files over their rawcontent service with streaming. I should probably write that in the documentation explicitly, since this is pretty esoteric

Reply:
Added documentation in 4eee4577